packages/ui/src/composites/wui-all-wallets-image/index.ts

```diff
@@ -0,0 +1,36 @@
+import { html, LitElement } from 'lit'
+import { customElement, property } from 'lit/decorators.js'
+import { globalStyles } from '../../utils/ThemeUtil'
+import styles from './styles'
+
+export interface WalletImage {
+  src: string
+  walletName: string
+}
+
+@customElement('wui-all-wallets-image')
+export class WuiAllWalletsImage extends LitElement {
+  public static styles = [globalStyles, styles]
+
+  // -- state & properties ------------------------------------------- //
+  @property({ type: Array }) public walletImages: WalletImage[] = []
+
+  // -- render ------------------------------------------------------- //
+  public render() {
+    const walletImagesHtml = this.walletImages.map(walletImage =>
+      this.renderWalletImage(walletImage.src, walletImage.walletName)
+    )
+
+    return html` <div>${walletImagesHtml}</div> `
```

Comment:
Do we need div here, perhaps we can style `:host` element directly? If so, this return can be simplified to

```
return this.walletImages.map(({src, alt})=> html`<wui-wallet-image size="inherit" src=${src} alt=${alt}></wui-wallet-image>`)
```